go/consensus/api/api.go

```diff
@@ -110,13 +109,13 @@ type ClientBackend interface {
 	// Note that an epoch is considered reached even if any epoch greater than
 	// the one specified is reached (e.g., that the current epoch is already
 	// in the future).
-	WaitEpoch(ctx context.Context, epoch epochtime.EpochTime) error
+	WaitEpoch(ctx context.Context, epoch beacon.EpochTime) error
 
 	// GetEpoch returns the current epoch.
-	GetEpoch(ctx context.Context, height int64) (epochtime.EpochTime, error)
+	GetEpoch(ctx context.Context, height int64) (beacon.EpochTime, error)
 
-	// EpochtimeConsensusParameters returns the epochtime consensus parameters.
-	EpochtimeConsensusParameters(ctx context.Context, height int64) (*epochtime.ConsensusParameters, error)
+	// BeaconConsensusParameters returns the beacon consensus parameters.
+	BeaconConsensusParameters(ctx context.Context, height int64) (*beacon.ConsensusParameters, error)
```

Comment:
This should probably be removed in favor of a beacon gRPC service. If not in this PR, please open an issue for it.

Reply:
I added filing an issue to the post-merge TODO.